tests/test_workers/test_tasks/test_build_create_empty_index.py

```diff
@@ -120,12 +121,99 @@ def test_handle_create_empty_index_request(
 @mock.patch('iib.workers.tasks.build_create_empty_index._cleanup')
 @mock.patch('iib.workers.tasks.build_create_empty_index.is_image_fbc')
 def test_handle_create_empty_index_request_raises(mock_iifbc, mock_c):
+    # Ensure that IIb raises error when fbc paramater is false
+    # but from_index provided is FBC
     mock_iifbc.return_value = True
     with pytest.raises(IIBError):
```

Comment:
May I suggest to modify this and add `match` parameter to check if expected error message was raised `with pytest.raises(IIBError, match=error_msg):`

Reply:
Added, Thanks for suggestion, ended up improving the test and optimizing the code :)